python/ngen_init_config/src/ngen/init_config/serializer_deserializer.py

```diff
@@ -70,3 +72,13 @@ class JsonSerializerDeserializer(ser.JsonSerializer, de.JsonDeserializer):
     """
 
     ...
+
+
+class GenericSerializerDeserializer(ser.GenericSerializer, de.GenericDeserializer, ABC):
+    """
```

Comment:
Why does this class explicitly subclass `ABC`? It's effectively already an abstract class via inheritance, and it doesn't have any of it's own abstract methods. Other than possibly obscure MRO reasons is this needed?

Reply:
Just for clarity. It is redundant. My assumption is that most use cases will subclass this instead of `ser.GenericSerializer` or `de.GenericDeserializer` directly.  Unless you read the docs it might not be entirely clear that you need to implement methods. Happy to get rid of it if you think its confusing. Thoughts?

Reply:
Python will tell you real quick if you don't implement them.  I would drop it.

Reply:
Just dropped it.